src/screens/Home/Home.js

```diff
@@ -138,6 +146,13 @@ function Home() {
     dispatch(fetchAllAccountsTotalBalancesAction());
   };
 
+  const onApps = async () => {
```

Comment:
```suggestion
  const onNativeIntegrationsLaunch = async () => {
```

`onApps` is too generic / broad...

Reply:
Okay, I'll change it.